### DISCUSSION TITLE
委托：洛朗展开

### DISCUSSION THREAD

**Discussion by zamsirs** [share121/inter-knot#5232] [其他]
关于洛朗展开这一块儿，我做题时发现好多题比如这两道，都是以原点为圆心化成z方分之一，却不是z方。我想不明白
![mmexport1726797918731](https://github.com/user-attachments/assets/b768d172-fc38-4fe2-a7f7-3398334f7d0f)

**Comment by theincrediblewwz** [comment]
因为提出来的z，z-2，模都是大于一的，如果不提出来的话，你看看写出来的级数，它每一项都是模大于一的东西，都不收敛了，提出来之后，分母就是1加上一个模小于1的东西，而模小于1时候，级数收敛是知道的

**Reply by zamsirs** [reply]
感谢佬

**Comment by Fluorine-z** [comment]
![img_326375153_1712326007779](https://github.com/user-attachments/assets/08a2798b-7e43-47fe-ae5a-34dd33278186)
我学化工的，看不懂

**Comment by Aha-follower** [comment]
啊？我就说上绳网能学到知识吧

**Reply by RWHwaxili** [reply]
要不哪天我试试发个高一政治教学帖？

**Comment by Muano2333** [comment]
这里是真能学到东西啊

**Comment by wasd040325** [comment]
学完已经忘记嘞😒
